filters/encrypt/filter.go

```diff
@@ -111,6 +111,21 @@ func (ef *Filter) Process(ctx context.Context, e *eventlogger.Event) (*eventlogg
 		return e, nil
 	}
 
+	var filtered bool
+	filterOps := DefaultFilterOperations()
+	for k, v := range ef.FilterOperationOverrides {
+		filterOps[k] = v
```

Comment:
I think you could combine this and the below for loop into one.

Reply:
ty!